bulk/ltc-grh-list-generator.vbs

```diff
@@ -122,14 +117,15 @@ objExcel.Visible = True
 Set objWorkbook = objExcel.Workbooks.Add()
 objExcel.DisplayAlerts = True
 
-excel_row = 2 
+excel_row = 2
 'Setting the first 4 col as worker, case number, name, and APPL date
-ObjExcel.Cells(1, 1).Value = "Worker"
+ObjExcel.Cells(1, 1).Value = "Worker #"
 ObjExcel.Cells(1, 2).Value = "MAXIS Case #"
-ObjExcel.Cells(1, 3).Value = "Client name"
-ObjExcel.Cells(1, 4).Value = "Facility name"
+ObjExcel.Cells(1, 3).Value = "Client Name"
+ObjExcel.Cells(1, 4).Value = "Facility Name"
+ObjExcel.Cells(1, 5).Value = "Vendor #"
 
-col_to_use = 5 'Starting with 5 because cols 1-4 are already used
+col_to_use = 5 'Starting with 6 because cols 1-5 are already used
```

Comment:
This should be 6, I think - from reading - otherwise you are going to write over the vendor column

Reply:
Good catch @C-Love, thank you!